lib/collectionspace/tools/xml.rb

```diff
@@ -187,74 +183,6 @@ def self.add_single_level_group_list(
         CSXML.add_group_list(xml, key, groups, list_suffix: list_suffix, group_suffix: group_suffix)
       end
 
-      # convenience method to build the following structure:
-      # topGroupList
```

Comment:
Since we are no longer treating a structured date value like a special hierarchy level, we don't need this method any more. I updated all uses of this one.